### DISCUSSION TITLE
Add manually curated bounties

### DISCUSSION THREAD

**Discussion by alxs** [solodit/feedback#39] [New Feature Requests]
Some protocols host bounty programs themselves, choosing to do without an intermediary such as Immunefi or Hats.finance. A community list of such programs can be found [here](https://github.com/JeffCX/collection-web3-bug-bounty). It would be great if Solodit added these (some of which may be outdated) and/or provided a community-based process to manage external bounties.

**Comment by hans-cyfrin** [comment]
Thank you, we will work on them soon.

**Comment by hans-cyfrin** [comment]
This has been implemented. Please check [here](https://github.com/solodit/solodit_content/blob/main/README.md#custom-bug-bounties).
Please feel free to contribute and add any other custom bug bounties.

**Comment by alxs** [comment]
Thanks! Those should probably be verified, AAVE for example no longer run their own bounty program.
